ivy/functional/frontends/paddle/tensor/math.py

```diff
@@ -210,6 +210,12 @@ def exp(x, name=None):
     return ivy.exp(x)
 
 
+@with_supported_dtypes({"2.5.0 and below": ("float32", "float64")}, "paddle")
+@to_ivy_arrays_and_back
+def expm1(x, name=None):
```

Comment:
Also, to merge the PR you will need to revert the changes in the `array_api_testing` folder. We only want to merge changes related to the function you are adding.